docs/integration/webapi.rst

```diff
@@ -228,6 +228,31 @@ Autofac introduces the new interfaces to allow you to concentrate on implementin
 
 Another reason for creating the internal attribute wrappers is to support the ``InstancePerRequest`` lifetime scope for filters. See below for more on that.
 
+Setting the Response in an Autofac Action Filter
+------------------------------------------------
+
+In the same way as with standard Web API filters,  you are able to set the HttpResponse object in the
```

Comment:
It looks like it's an `HttpResponseMessage` not an `HttpResponse` [from the code](https://github.com/aspnet/AspNetWebStack/blob/ebef5b7d821b64ed5c48765d0caf6ce8a9bcfaf5/src/System.Web.Http/Controllers/HttpActionContext.cs#L82).

Reply:
There's another instance of that below.

Reply:
Corrected the name, added backticks.